x-pack/plugins/enterprise_search/public/applications/shared/kibana/kibana_logic.ts

```diff
@@ -6,13 +6,15 @@
 
 import { kea, MakeLogicType } from 'kea';
 
+import { FC } from 'react';
 import { ApplicationStart, ChromeBreadcrumb } from 'src/core/public';
 
 export interface IKibanaValues {
   config: { host?: string };
   navigateToUrl: ApplicationStart['navigateToUrl'];
   setBreadcrumbs(crumbs: ChromeBreadcrumb[]): void;
   setDocTitle(title: string): void;
+  renderHeaderActions(HeaderActions: FC): void;
```

Comment:
TIL typing this way 💯

Reply:
It felt weird importing all of React just for React.FC so I destructured it out. Totally agreed it looks weirdly naked though haha